doc/nrf/gs_programming.rst

```diff
@@ -61,6 +61,12 @@ You can download it from the following links:
 #. Build and flash your project.
    The required steps differ depending on if you build a single application or a multi-image project (such as the nRF9160 samples, which include :ref:`SPM <secure_partition_manager>`).
 
+   .. important::
```

Comment:
Does this belong in this commit?

Reply:
Yes - because it was moved out of the sample documentation (that explained how to flash sample + SPM).